src/test/java/uk/gov/companieshouse/api/accounts/validation/AccountsValidatorTest.java

```diff
@@ -136,6 +136,38 @@ void setUp() throws DataException {
         ReflectionTestUtils.setField(validator, MANDATORY_ELEMENT_MISSING_KEY, MANDATORY_ELEMENT_MISSING);
     }
 
+    @Test
+    @DisplayName("Validate Submission - single year filer, successful, no errors")
+    void validateSubmissionSYFNoErrorsFound() throws DataException, ServiceException {
+
+        Errors emptyErrors = new Errors();
+
+        when(smallFullService.find(COMPANY_ACCOUNTS_ID, request)).thenReturn(smallFullResponseObject);
+        when(smallFullResponseObject.getData()).thenReturn(smallFull);
+
+        when(currentPeriodTnClosureValidator.validate(any(String.class), any(SmallFull.class),
+                any(HttpServletRequest.class), any(Errors.class))).thenReturn(emptyErrors);
+        when(previousPeriodTnClosureValidator.validate(any(String.class), any(SmallFull.class),
+                any(Transaction.class), any(HttpServletRequest.class), any(Errors.class)))
+                .thenReturn(emptyErrors);
+
+        when(currentPeriodService.find(COMPANY_ACCOUNTS_ID, request)).thenReturn(cpResponseObj);
+        when(cpResponseObj.getData()).thenReturn(currentPeriod);
+        when(currentPeriod.getBalanceSheet()).thenReturn(currentPeriodBalanceSheet);
+
+        when (companyService.isMultipleYearFiler(transaction)).thenReturn(false);
+
+        when(stocksTnClosureValidator
+                .validate(COMPANY_ACCOUNTS_ID, smallFull, transaction, request, emptyErrors, currentPeriodBalanceSheet, null))
```

Comment:
More readable if you use a local var with a meaningful name, assign that null, & pass it in. A bit verbose, but more readable.

Reply:
good point, will add that now :)

Reply:
After tempting to add it, intellij recommended I change it back to null. Discussed and we will keep it null for now.